labs/angular-1/add/add.js

```diff
@@ -0,0 +1,11 @@
+angular.module("todoList").component("addItem", {
+  templateUrl: "add/add.html",
+  controller: ['itemEntry', function(itemEntry) {
```

Comment:
This dependency list is for *services*, not for other *components*. No need to inject `itemEntry`, here — but someday you may inject an `ItemService`